### PR TITLE
README: add examples from products of different companies

### PR DESCRIPTION
**Pull request by myungjoo**
The companies, Fainders.AI, NXP, PrintNanny, openncc, and Klleon.IO, are using nnstreamer for their products and services.
With their permission, add their demonstration video clips to the readme.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: Documentation/products.md [hunk 1]
```diff
@@ -20,10 +20,14 @@
...
-- KLleon: video processing.
-- NXP Semiconductors: edge-AI developing tool.
-- OpenNCC: edge-AI developing tool.
+| Company |  |  |
+| :---: | :---: | --- |
+| [fainders.ai](https://fainders.ai) | ![fainders.ai.demo.store](media/FaindersAI_Store.webp) | Unmanned retail store |
+| [klleon](https://klleon.io) | ![klleon.demo.speech](media/Klleon_Speech.webp) | Generate speech video from pictures and texts. |
+| [OpenNCC](https://openncc.com) | ![openncc.demo.facemask](media/OpenNCC_Mask.webp) | Edge-AI cameras and real-time AI service framework |
+| [PrintNanny](https://printnanny.ai) | ![printnanny.demo.printer](media/PrintNanny.webp) | 3D printer monitoring service |
+| [NXP](https://nxp.com) | [i.MX Machine Learning User's Guide](https://www.nxp.com/docs/en/user-guide/IMX-MACHINE-LEARNING-UG.pdf) | Vision pipeline with NNStreamer |
```
Comment (anyj0527) [inline Documentation/products.md:29]: https://github.com/NXPmicro/nxp-nnstreamer-examples could be referred.

Comment (myungjoo) [reply Documentation/products.md:29]: Those appear really good! Thanks!

**Review by DonghakPark — APPROVED** [review]
LGTM!!

**Review by niley7464 — APPROVED** [review]
LGTM :)

**Review by anyj0527 — APPROVED** [review]
LGTM 👍

**Pull request merged**